android/apollo/src/com/andrew/apollo/MusicPlaybackService.java

```diff
@@ -1345,70 +1323,14 @@ public void openFileResult(boolean result) {
      *              otherwise.
      * @return The next position to play.
      */
-    private int getNextPosition(final boolean force) {
+    private int getNextPosition(final boolean force, final boolean shuffleMode) {
```

Comment:
rename parameter `shuffleMode` to `shuffleEnabled` or `shuffleOn` (whatever you prefer)